File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.py
class Solution(object):
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ans = 0
        
        for num in nums:
            cnt = 0
            while num > 0:
                num /= 10
                cnt += 1
            if cnt % 2 == 0:
                ans += 1
                
        return ans
        

File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        // 자리수가 짝수인
        // String으로
        int n = 0;
        for(int i = 0; i < nums.length; i++) {
            String s = Integer.toString(nums[i]);
            if(s.length() % 2 == 0)
                n++;
        }
        return n;
    }
}